osbrain/agent.py

```diff
@@ -1219,7 +1217,7 @@ def shutdown(self):
 
     def kill(self):
         self.close_sockets()
-        self.kill_agent = True
+        self._pyroDaemon.shutdown()
```

Comment:
Good question.

We are not setting that attribute, but Pyro does:

https://pyro4.readthedocs.io/en/stable/servercode.html#attributes-added-to-pyro-objects

We assume if the `Agent` is running, then it is being served with Pyro (I think it is a sane assumption). If it is not running, then we shouldn't be calling `.kill()` anyway.